libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardView/DashboardView.tsx

```diff
@@ -173,43 +175,47 @@ export const DashboardView: React.FC<IDashboardViewProps> = ({
                 <ColorPaletteProvider palette={colorPalette}>
                     <AttributesWithDrillDownProvider attributes={drillDownAttributes}>
                         <DashboardAlertsProvider alerts={alertsData}>
-                            <ScheduledMailDialog
-                                backend={backend}
-                                workspace={workspace}
-                                locale={effectiveLocale}
-                                dashboard={dashboard}
-                                filters={applyFiltersToScheduledMail ? sanitizedFilters : undefined}
-                                onSubmit={onScheduledMailDialogSubmit}
-                                onSubmitSuccess={onScheduledMailSubmitSuccess}
-                                onSubmitError={onScheduledMailSubmitError}
-                                onCancel={onScheduledMailDialogCancel}
-                                onError={onError}
-                                isVisible={isScheduledMailDialogVisible}
-                            />
-                            {isFluidLayoutEmpty(dashboardData.layout) ? (
-                                <EmptyDashboardError ErrorComponent={ErrorComponent} />
-                            ) : (
-                                <DashboardRenderer
+                            <DashboardViewIsReadOnlyProvider isReadOnly={isReadOnly ?? false}>
```

Comment:
Nitpick: isn't more proper place for default on line 49 (isReadonly = false)?

Reply:
It most definitely is, updated